crates/egui_plot/src/lib.rs

```diff
@@ -854,7 +854,7 @@ impl Plot {
             PlotMemory::load(ui.ctx(), plot_id)
         }
         .unwrap_or_else(|| PlotMemory {
-            auto_bounds: true.into(),
+            auto_bounds: default_auto_bounds,
```

Comment:
was wondering about this one on the previous PR that added `default_auto_bounds`. Got me I should have commented 😄